spec/fixtures/eurofxref-daily.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah, they are manually downloaded from the source. I thought that it is easier to simply download static xml files and use them in tests because if their format or structure changes we will need to alter code anyway.